datafusion/core/benches/map_query_sql.rs

```diff
@@ -33,8 +34,15 @@ mod data_utils;
 
 fn build_keys(rng: &mut ThreadRng) -> Vec<String> {
     let mut keys = vec![];
-    for _ in 0..1000 {
-        keys.push(rng.random_range(0..9999).to_string());
+    let mut seen = HashSet::with_capacity(1000);
```

Comment:
We could also make `keys` a HashSet and just keep inserting into it until it reaches 1000 instead of having both `keys` and `seen`